.kokoro/samples-test.sh

```diff
@@ -20,10 +20,15 @@ export NPM_CONFIG_PREFIX=/home/node/.npm-global
 
 # Setup service account credentials.
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
-export GCLOUD_PROJECT=node-gcloud-ci
+export GCLOUD_PROJECT=long-door-651
```

Comment:
Is this a problem?

Reply:
these are from synth templates - they are overridden in `pre-samples-test.sh`